src/Framework/Conventions/KineVar.h

```diff
@@ -53,6 +53,8 @@ typedef enum EKineVar {
   kKVv,
   kKVSelPn,
   kKVSelv,
+  kKVQ0,
+  kKVQ3,
```

Comment:
You also need to set the Selected values here. 
Not only the running values.
And then set them in the event generator (see the comment about rubbish casting)
Add the words too at the end of this file